contents/python/introduction-to-python.md

```diff
@@ -12,6 +12,23 @@ For absolute beginners, check out [this video](https://www.youtube.com/watch?v=N
 
 Both newbies and experienced programmers can also benefit from [The Python Tutorial](https://docs.python.org/3/tutorial/index.html), which aims to introduce readers to Python's unique features and style.
 
+## Python 2 vs Python 3
+
+Despite obvious similarities in two Python versions, Python 3's intentional backward incompatibility makes choosing to learn Python 3 over Python 2 a tough problem for both new and experienced Python programmers. Some concerns include the lack of popular Python 2 packages in Python 3 as well as changes in some Python built-in libraries that might break existing systems. One example is that a simple `print 'Hello World'` that runs perfectly in Python 2 will cause a syntax error in Python 3.
+
+Read more in this Digital Ocean's [post](https://www.digitalocean.com/community/tutorials/python-2-vs-python-3-practical-considerations-2) to understand this conundrum of choosing between Python 2 and Python 3 better. If you choose to learn both Python 2 and Python 3, take a look at some of these important [changes](https://www.geeksforgeeks.org/important-differences-between-python-2-x-and-python-3-x-with-examples/) to avoid gotchas due to version differences.
+
+Overall, it is recommended to learn Python 3 as a beginner today as it has been [10 years since Python 3 debutted](https://www.python.org/download/releases/3.0/). This stance is also [supported](http://python-notes.curiousefficiency.org/en/latest/python3/questions_and_answers.html#why-is-python-3-considered-a-better-language-to-teach-beginning-programmers) by core members of Python. Moreover, unlike the early days of Python 3's release, most of the popular packages from Python 2 is already supported in Python 3. Since Python 2 is a legacy language while Python 3 is in active development, it would be better to learn Python 3 today.
```

Comment:
- `debutted` -> `debuted`
- `most of the popular packages` -> `most popular packages` or `many popular packages`
- `is already supported in Python 3` -> `now support Python 3 as well`